src/utils/name-utils.ts

```diff
@@ -0,0 +1,13 @@
+import { get } from 'lodash';
+import { BoundaryLayerProps } from '../config/types';
+
+export function getLocationName(
```

Comment:
This function should have a docstring
`/**
...
**/`

Reply:
done. Can you check the docstring to make sure what i wrote makes sense?